src/vs/editor/contrib/referenceSearch/referenceSearch.ts

```diff
@@ -197,6 +212,16 @@ KeybindingsRegistry.registerCommandAndKeybindingRule({
 	handler: closeActiveReferenceSearch
 });
 
+KeybindingsRegistry.registerCommandAndKeybindingRule({
+	id: 'openReferenceToSide',
+	weight: KeybindingsRegistry.WEIGHT.editorContrib(),
+	primary: KeyMod.CtrlCmd | KeyCode.Enter,
+	mac: {
+		primary: KeyMod.WinCtrl | KeyCode.Enter
```

Comment:
Why not `cmd` on macs? We use that in the explorer and elsewhere

Reply:
@jrieken you must have changed the keybindings then, the default is `Ctrl+Enter` on mac for Explorer, Search and Problems view:

![image](https://user-images.githubusercontent.com/900690/34733143-a2bfd8ce-f567-11e7-9787-492e869bcd3a.png)

Reply:
true story...